spec/models/group_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
+  subject do
+    described_class.new(
+      id: 1,
+      name: 'Education',
+      icon: 'picture.png',
+      user_id: 1,
+      created_at: '29/04/23',
+      updated_at: '30/04/23'
+    )
+  end
+
+  it 'is valid with valid attributes' do
+    expect(subject).to be_valid
+  end
+

```

Comment:
- [ ] I think two tests here is failing so please fix the code to make your test work as expected since here I think you have validations in your group model that didn't let the test pass since you put the icon should be present but when a user create a new category you put some images in a drop sown menu so user can choose one of them but If you make it as input to let the user enter any image he want so that would let your test pass I think so try it 🤩🔥💯

![image](https://user-images.githubusercontent.com/58553711/235318538-d6f6bed1-9799-4f3f-8b43-a413fd8a62bd.png)

Reply:
Hi @Reem-lab, 
All the tests are passing at my end so can you verify it again?
![image](https://user-images.githubusercontent.com/105938633/235321773-b391168f-fb9f-4c54-a369-ad835b05c88e.png)

Reply:
- [ ] Hi @elarhadu , Ok I will check it

Reply:
Okay @Reem-lab. Can you also check for the other changes you requested for?